doc/getting-started/openshift-ai/README.md

```diff
@@ -0,0 +1,85 @@
+# Getting started with ai-telemetry in OpenShift AI
+
+## Access our project pods in the NERC OpenShift Console
+
+- Log in to the [NERC OpenShift Console here](https://console.apps.shift.nerc.mghpcc.org/k8s/ns/ai-telemetry-cbca60/core~v1~Pod). 
+
+## Obtain a CLI token to access project resources in the terminal
+
+- Click on your username in the top right, and select [ Copy login command ]. 
+- Paste the command into your terminal. 
+- Make sure you are on the `ai-telemetry-cbca60` project
+
+```bash
+oc project ai-telemetry-cbca60
+```
+
+## Set up an OpenShift AI Workbench
+
+- Visit the [OpenShift AI dashboard in NERC](https://rhods-dashboard-redhat-ods-applications.apps.shift.nerc.mghpcc.org/projects/ai-telemetry-cbca60)
+- Click the [ Create Workbench ] button. 
+- Give your workbench a unique name for you like your GitHub username. 
+- Image selection: VSCode IJava java-17-openjdk
+- Container size: Small
+- Select "Create new persistent storage"
+  - Name: Use the name of your workbench
+  - Persistent storage size: 20Gi or greater
+- Click [ Create workbench ]
+
+## Set up a rolebinding for the default service account to access other pods in namespace: 
+
+Run these commands in your computer's terminal with a current token. 
+First set an environment variable for your workbench name: 
+
+```bash
+OPENSHIFT_AI_WORKBENCH=computate
+```
+
+Run these commands to create roles and rolebindings for edit access to the project resources in OpenShift from your OpenShift AI Workbench. 
+
+```bash
+oc create rolebinding $OPENSHIFT_AI_WORKBENCH-edit --clusterrole=edit \
+  --serviceaccount=$(oc project -q):$OPENSHIFT_AI_WORKBENCH
```

Comment:
Is this rolebinding redundant? We're already creating one below.

Reply:
We talked about how it's not redundant, because this is an edit rolebinding for the service account, and the one below is to edit roles and rolebindings which are not provided by edit.